_sass/_customstyles.scss

```diff
@@ -334,6 +334,10 @@ img.screenshotSmall {
     max-width: 300px;
 }
 
+img.catalog {
```

Comment:
Let's create a new _samples_catalog.scss and put all the samples styles there. I've found it makes it easier to manage when we separate out styles by function.